app/views/admin/comments/index.html.haml

```diff
@@ -0,0 +1,24 @@
+%h1 Comments
+%br
+.row
+  .col-md-12
+    %ul.nav.nav-tabs#commentsTable
+      %li.active
+        %a{:href=>"#unread_comments", "data-toggle"=>"tab"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace